metaflow/plugins/metadata/service.py

```diff
@@ -357,6 +447,8 @@ def _request(cls, monitor, path, data=None, retry_409_path=None):
                         raise
                 resp = None
             else:
+                if return_raw_resp:
```

Comment:
The existing `_request()` cannot be easily extended with additional retry and failure handling logic as needed for tag mutation.  For now we add an escape hatch so tag mutation can do its own thing.  In future we should consider refactoring `_request` altogether to provide a cleaner abstraction here (e.g. setting common headers yes, retry policies will depend, etc).